jupyter_server/serverapp.py

```diff
@@ -782,7 +778,7 @@ def _token_default(self):
     def _token_changed(self, change):
         self._token_generated = False
 
-    password = Unicode(u'', config=True,
+    password = CUnicode(u'', config=True,
```

Comment:
With python 3, we don't need the u prefix here and elsewhere
```suggestion
    password = CUnicode('', config=True,
```